### PR TITLE
Adding finally callbacks

### PR DESCRIPTION
**Pull request by unlucio**

### CODE REVIEW

**Comment by unlucio** [comment]
right, I added the fault :P

**Comment by unlucio** [comment]
it gets tricky with scopes because of the way promises are implemented inside V8

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ independent such as "free form dependencies": it would be nice to be able to inc
 ## List:
 
 - [Props](https://github.com/unlucio/promiseKit/blob/master/props): Kind of like `all()` but it receives and return an object (dictionatry) instead of an array.
+- [Props](https://github.com/unlucio/promiseKit/blob/master/finally): Just like in a try/catch clausule, functions registered as "finally" will be called at the end of a promise cicle.
```
Comment (MicheleBertoli) [inline README.md]: s/Props/Finally

File: finally/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    fcb();
+  });
+};
+
+Promise.prototype.then = function(callback, errorCb) {
+  this.finallyCb = this.finallyCb || [];
+  var self = this;
+
+  function thenCallback(result) {
+    var cbReturn = callback(result);
```
Comment (claudioc) [inline finally/index.js:23]: Using a `callback` without testing if it's runnable

File: finally/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  next.finallyCb = this.finallyCb;
+
+  return next;
+};
+
+Promise.prototype.catch = function(callback) {
+  var self = this;
+
+  function erroCallback(error) {
+    callback(error);
```
Comment (claudioc) [inline finally/index.js]: Same here (no test on the callback type)

File: finally/index.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Promise.prototype.finally = function(callback) {
+  this.finallyCb = this.finallyCb || [];
+  this.finallyCb.push(callback);
+  return this;
+};
+
+Promise.prototype.fireFinally = function() {
+  this.finallyCb.forEach(function(fcb) {
+    fcb();
```
Comment (claudioc) [inline finally/index.js:14]: Using a callback without testing if it's runnable

**Pull request merged**